### PR TITLE
Build cross docs

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,31 @@ etcdadm is a command-line tool for operating an etcd cluster. It makes it easy t
...
 ## Getting Started
 
-### Building
+### Quick start: How to quickly create a production etcd cluster using etcdadm
 
-```
-go get -u sigs.k8s.io/etcdadm
-```
+- Build a release of etcdadm.  Assuming your nodes are linux, run `GOOS=linux make container-build`.
+- copy the created `etcdadm` file it to every node on your cluster.
```
Comment (ojmhetar) [inline README.md]:
```suggestion
- copy the created `etcdadm` file to every node on your cluster.
```

File: README.md [hunk 2]
```diff
@@ -22,13 +22,31 @@ etcdadm is a command-line tool for operating an etcd cluster. It makes it easy t
...
+- Then run `go get -u sigs.k8s.io/etcdadm`
+
+#### As a user
+- Build in docker, its easy ! Just run `make container-build`
+- The outputted binary will be natively runnable on your mac or linux machine.
 
-### Creating a new cluster
+### Contributing
+- Follow the above directions, 
+- cd $GOPATH/github.com/platform9/etcdadm
```
Comment (ojmhetar) [inline README.md]:
```suggestion
- cd $GOPATH/sigs.k8s.io/etcdadm
```

File: README.md [hunk 3]
```diff
@@ -22,13 +22,31 @@ etcdadm is a command-line tool for operating an etcd cluster. It makes it easy t
...
+- On successive nodes, just run "Adding a member" instructions below.
+
+### Building, Contributing
+
+#### As a developer
+- Setup Golang: The fastest way to do this is using https://github.com/travis-ci/gimme
+- Then run `go get -u sigs.k8s.io/etcdadm`
+
+#### As a user
+- Build in docker, its easy ! Just run `make container-build`
```
Comment (ojmhetar) [inline README.md]:
```suggestion
- Build in docker, it's easy ! Just run `make container-build`
```

File: README.md [hunk 4]
```diff
@@ -37,7 +55,11 @@ go get -u sigs.k8s.io/etcdadm
 etcdadm init
 ```
 
-### Adding a member
+You now have an etcd 'cluster' which is usable, of size 1.
+
+### Step 2: Adding a member
+
+Although not necessary, its obviously a good idea to have more then 1 node in your cluster.
```
Comment (ojmhetar) [inline README.md]:
```suggestion
Although not necessary, it's obviously a good idea to have more than 1 node in your cluster.
```

File: Gopkg.lock [hunk 5]
```diff
@@ -158,6 +158,20 @@
   pruneopts = "UT"
   revision = "e89373fe6b4a7413d7acd6da1725b83ef713e6e4"
 
+[[projects]]
```
Comment (ojmhetar) [inline Gopkg.lock]: Your Gopkg.lock is out of date, please run a `dep ensure` to fix. (This file shouldn't be in the PR for a readme change.)

**Comment by jayunit100** [comment]
moved README out to another PR for later, this is just the makefile updates and vendoring stuff thanks @ojmhetar

**Comment by dlipovetsky** [comment]
@jayunit100 Thanks for the Makefile improvements!

What it is purpose of vendoring etcdadm?

**Comment by ojmhetar** [comment]
Sorry, my intention was not to ask to vendor etcdadm, but instead to remove `Gopkg.lock` from the original commit which was only readme changes.

**Comment by jayunit100** [comment]
sure, yeah ill clean out this PR so its just one change

**Comment by jayunit100** [comment]
oh, i see, i think i had to vendor for a compile issue, ill see if i can reproduce in a separate issue when i get time to look at it.

**Pull request closed without merging**